lib/colorls/flags.rb

```diff
@@ -141,6 +142,9 @@ def add_common_options(options)
       options.on('-f', '--files', 'show only files')                      { @opts[:show] = :files }
       options.on('--gs', '--git-status', 'show git status for each file') { @opts[:git_status] = true }
       options.on('--report', 'show brief report')                         { @show_report = true }
+      options.on('--indicator-style=[STYLE]', String, 'append indicator to entry names') do |style|
+        @opts[:indicator_style] = style
+      end
```

Comment:
There should be some help which values are valid for `STYLE`. And only those should be accepted (similar to the `--sort` option).

Also, if `--indicator-style=slash` is supported, we should also support `-p` as a shorthand option for it.

Reply:
Thank you for your comment!!
I added a description that only `none` is valid for `STYLE`.

For the `--indicator-style=slash` option, colorls append a backslash to directory name by default. 
So, I think it is unnecessary to support `--indicator-style=slash`.

Reply:
> For the `--indicator-style=slash` option, colorls append a backslash to directory name by default.
> So, I think it is unnecessary to support `--indicator-style=slash`.

Since colorls should support all the options of GNU ls, it should support this option too. Also, think about defining command line aliases with some specific option, like `alias ll='colorls --indicator-style=none -l'` you still should have a chance to override these options set in the alias on the command line, e.g. running `ll -p'.